rfcs/RFC-0015-onion-services/assets/hopr-protocol-summary.md

```
@@ -0,0 +1,333 @@
+Shared secrets: standard SPHINX chain. For each hop `i`,
+`SharedSecret_i = KDF("HASH_KEY_SPHINX_SECRET", Coeff × P_i, P_i)`, Alpha and
+Coeff blinded by `B_i = KDF("HASH_KEY_SPHINX_BLINDING", ·, Alpha)`.
+Replay protection: per-packet `ReplayTag = KDF("HASH_KEY_PACKET_TAG",
+SharedSecret_i)`; nodes reject previously seen tags.
+
+Header prefix byte per hop: bits [7–5] version (`001`), bit 4 `NoAckFlag`,
+bit 3 `ReplyFlag`, bits [2–0] hop position.
+
+### 2.3 Payload format
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)